frontend/components/product_markup.js

```diff
@@ -0,0 +1,46 @@
+/*
+<div key={ i } className='products-summary'>
```

Comment:
no need to leave things commented out, we can get it from previous version of the code if we need it